src/platform/packages/shared/serverless/settings/observability_project/index.ts

```diff
@@ -34,4 +34,5 @@ export const OBSERVABILITY_AI_ASSISTANT_PROJECT_SETTINGS = [
   settings.OBSERVABILITY_AI_ASSISTANT_SIMULATED_FUNCTION_CALLING,
   settings.OBSERVABILITY_AI_ASSISTANT_SEARCH_CONNECTOR_INDEX_PATTERN,
   settings.AI_ASSISTANT_PREFERRED_AI_ASSISTANT_TYPE,
+  settings.AI_ANONYMIZATION_SETTINGS,
```

Comment:
if this needs to be shown in Serverless Elasticsearch too then you should add this to the [search_project](https://github.com/elastic/kibana/blob/main/src/platform/packages/shared/serverless/settings/search_project/index.ts) settings as well.